frontend/src/components/Table/Seat/Seat.tsx

```diff
@@ -14,14 +14,13 @@ type SeatProps = {
 export default function Seat({ seatNumber, chipCount, socket }: SeatProps) {
   const playerSit = useCallback(async (event: React.MouseEvent) => {
     const payload = { selectedSeatNumber: event.currentTarget.id, socketId: socket.id };
-
     const result = await FetchFasade.post<PlayerSitPayload, PlayerSitResult>('/api/actions/playerSit', payload);
-
-    if (result.ok) {
-      console.log(result.getValue());
-    } else {
-      console.log('error', result.errorMessage);
-    }
+    console.log('result ok check',result);
```

Comment:
as you mentioned in Slack, we aren't expecting anything so we should change the PlayerSitResult to simply return an ok true.

Reply:
Right ok. I think I got tripped up partly by the fact the ok boolean is located in the value of the response object.

So for both ok true and ok false the response shape is still 

ResultSuccess {ok: true...., value: {ok:...}}